### DISCUSSION TITLE
Quire install on linux?

### DISCUSSION THREAD

**Discussion by nastarovia** [thegetty/quire#334] [Get Help]
Hi, has anyone been able to successfully install quire on linux? I have tried different methods, but have failed miserably (the installation instructions page mentions a linux support, but then no specific instructions are given).... Any ideas?

**Comment by Erin-Cecele** [comment]
Hi @nastarovia. Thank you for reaching out. Our apologies, as the Linux documentation is a bit out of date. Our ability to support Linux has somewhat decreased over the years as Quire has rapidly advanced. However, we will be looking to improve Linux support with future versions of Quire. So please stay tuned!
